mbedtls-sys/build/cmake.rs

```diff
@@ -46,25 +46,20 @@ impl super::BuildConfig {
             cmk.define("CMAKE_TRY_COMPILE_TARGET_TYPE", "STATIC_LIBRARY");
         }
 
-        let mut dst = cmk.build();
+        let dst = cmk.build();
 
-        dst.push("lib");
         println!(
             "cargo:rustc-link-search=native={}",
-            dst.to_str().expect("link-search UTF-8 error")
+            dst.join("lib").to_str().expect("link-search UTF-8 error")
         );
 
-        println!("cargo:rustc-link-lib=mbedtls");
-        println!("cargo:rustc-link-lib=mbedx509");
-        println!("cargo:rustc-link-lib=mbedcrypto");
+        println!("cargo:rustc-link-lib=static=mbedtls");
+        println!("cargo:rustc-link-lib=static=mbedx509");
+        println!("cargo:rustc-link-lib=static=mbedcrypto");
 
         println!(
             "cargo:include={}",
-            ::std::env::current_dir()
-                .unwrap()
-                .join(&self.mbedtls_include)
```

Comment:
Not sure if this var is used anywhere else.

Reply:
It doesn't. This part is just trying to provide header files location to dependent crates that can be read by `DEP_MBEDTLS_INCLUDE` environment variable. Either from the source directory or alternatives (out_dir). and from dependent crates, it does not matter as long as it has read access.
Cargo uses a flat model and you have full access to the file system but Bazel sandboxing requires some practices to follow for more compatibility.

Reply:
So then the var can be deleted?

Reply:
`self.mbedtls_include` is still being used by `Bindgen` but I'll check if headers are also accessible in there, Since it's running after the CMake step.
```
fn main() {
    let cfg = BuildConfig::new();
    cfg.create_config_h();
    cfg.print_rerun_files();
    cfg.cmake();
    cfg.bindgen();
}
```

Reply:
Now, Only CMake depends on the source path and other components are using OUT_DIR content.